### ISSUE TITLE
Studio: statistical tests

### ISSUE THREAD

**Issue by sheryjoe** [SCIInstitute/ShapeWorks#513]
From Amy Lenz:

Include statistical analyses in Studio for example: 1) Parallel analysis for containing significant PCA modes of variation.  2) Within these significant modes, PCA loading values compared between two groups, a student's T-test with Finner's adjustment for multiple comparisons would be great, and 3) Hotelling's T^2 test for determining significant shape differences between two groups.

**Comment by sheryjoe** [comment]
@akenmorris I am bringing a biostatistician on the SW team. He can provide us with python functions for these tests. How easy to integrate this within Studio? As we plan to support other languages besides python, can we consider implementing these tests in C++? other thoughts?

**Comment by akenmorris** [comment]
I think they should be done in C++ with APIs provided to other languages.  It will be very cumbersome to have all tools rely on calling out to python for core functionality.

**Comment by sheryjoe** [comment]
Agreed. It would help us in moving them to C++ if he implements them using native python without relying on specific libraries unless you are aware of C++ libraries that have such tests implemented.

**Comment by sheryjoe** [comment]
@akenmorris let's meet with Nan to discuss this issue in addition to multiple comparisons and shae-feature covariation.
